### ISSUE TITLE
"pickValues" Overload Syntax for Values

### ISSUE THREAD

**Issue by machineghost** [jashkenas/underscore#2445]
Currently Underscore has a `pick` method which returns a specific set of keys/values (in object form), and it has a `values` method which returns all values.  However, there is no overlapping function.  In other words, there's no way to select a specific set of values from an object without combining the two functions:

```
_.chain({foo: 1, bar: 2, baz: 3}).pick('foo', 'baz').values().value(); // returns [1, 3]
```

Wouldn't it be great if you could do all of that with a single Underscore function?  And better yet, what if you could do it without even adding any new functions to the library?

In other words, wouldn't it be great if you could do:

```
_.values({foo: 1, bar: 2, baz:3}, 'foo', 'bar'); // returns [1, 3]
```

Of course a new `pickValues` method would work also, but this overload of `values` seems very natural; even with it `values` still returns the values of an object, it just returns a more limited sub-set.  And because `values` has only ever taken one argument, it would be perfectly safe to add support for another argument, to enable this new functionality.

Just a suggestion.

**Comment by jdalton** [comment]
For reference lodash has [_.at](https://lodash.com/docs#at).
```js
_.at({ 'a': 1, 'b': 2, 'c': 3}, ['a', 'c'])
// [1, 3]
```

**Comment by jgonggrijp** [comment]
Closing for the same reason as #2532.
